x-pack/plugin/logsdb/src/main/java/org/elasticsearch/xpack/logsdb/patterntext/PatternTextFieldMapper.java

```diff
@@ -48,17 +48,17 @@
  * A {@link FieldMapper} for full-text log fields that internally splits text into a low cardinality template component
  * and high cardinality argument component. Separating these pieces allows the template component to be highly compressed.
  */
-public class PatternedTextFieldMapper extends FieldMapper {
+public class PatternTextFieldMapper extends FieldMapper {
 
-    public static final FeatureFlag PATTERNED_TEXT_MAPPER = new FeatureFlag("patterned_text");
+    public static final FeatureFlag PATTERN_TEXT_MAPPER = new FeatureFlag("pattern_text");
     private static final NamedAnalyzer STANDARD_ANALYZER = new NamedAnalyzer("standard", AnalyzerScope.GLOBAL, new StandardAnalyzer());
 
     /**
      * A setting that indicates that patterned text fields should disable templating, usually because there is
```

Comment:
Missed the comment here
```suggestion
     * A setting that indicates that pattern text fields should disable templating, usually because there is
```